studies/s_integrate_out_noise.nb

```diff
@@ -0,0 +1,5877 @@
+(* Content-type: application/vnd.wolfram.mathematica *)
```

Comment:
Should we set up a separate repo for this stuff? It all seems to be cluttering up the pints repo a bit.

Reply:
Happy to have one, yep.